apigw-comprehend/README.md

```diff
@@ -0,0 +1,68 @@
+# API Gateway REST API to Amazon Comprehend
+
+This template creates an API Gateway REST API integration with Amazon Comprehend.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/apigw-rest-api-comprehend.
+
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd serverless-patterns/apigw-rest-api-comprehend
+    ```
+1. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy --guided
+    ```
+1. During the prompts:
+    * Enter a stack name
+    * Enter the desired AWS Region
+    * Allow SAM CLI to create IAM roles with the required permissions.
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+1. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## Testing
+
+To test the endpoint first send data using the following command. Be sure to update the API Endpoint, Job Definition and the Job Queue from the Cloudformation Stack Output.
+
+```
+curl --location --request POST '[YOUR API ENDPOINT]' --header 'Content-Type: application/json' \
+--data-raw '{
+        "LanguageCode": "en",
+        "Text": "Bob lives in Seattle. He is a software engineer at Amazon."
+    }'
+```
+
+You can also run the following commands on a terminal prompt to get the API Endpoint, Job Queue and Job definition.
+
+**API Endpoint**
+```
+aws cloudformation describe-stacks --stack-name [YOUR STACK NAME] --query "Stacks[0].Outputs[?OutputKey=='ApiEndpoint'].OutputValue" --output text
```

Comment:
"Stacks[0].Outputs[?OutputKey=='ComprehendApiEndpoint']

Reply:
It seems the file didn't get saved while editing in VS Code. Changes updated